package.json

```diff
@@ -9,11 +9,11 @@
     "coverage": "jest -c jest.config.js --collect-coverage",
     "eslint": "eslint --config .eslintrc.js \"**/*.js\"",
     "eslint-fix": "eslint --config .eslintrc.js \"**/*.js\" --fix",
-    "lint": "npm run eslint",
+    "lint": "pnpm eslint",
     "prettier": "prettier --write \"./**/*.{md,jsx,json,html,css,js,yml}\"",
     "prettier-check": "prettier --check \"./**/*.{md,jsx,json,html,css,js,yml}\"",
     "prepare": "husky install",
-    "pre-commit": "pretty-quick --staged && npm run lint"
+    "pre-commit": "pretty-quick --staged && pnpm run lint"
```

Comment:
You don't need `run` with `pnpm`

Reply:
Let's add a `preinstall` script like https://github.com/Seneca-CDOT/telescope/blob/master/package.json#L35